ArkEcosystem/laravel-ui
# fix:  navbar disable scroll lock only on mobile devices

## Summary
https://app.clickup.com/t/jtbwwn

The scroll lock used to solve some issues on the menu scrolling on mobile devices was causing a shift on Windows-based browsers due to the scrollbar disappearing.

On lg screen that lock is not really necessary (it can even be annoying) so this PR changes the navbar script to disable the scrolling only on mobile (< 640px) devices. 




To test merge this to msq and

- Interact with the menu on your desktop browser and different screen sizes
- To test the shift on windows browser you will need windows ☠️


Note: There is an issue on the menu on ios devices (you cannot scroll enough to see the bottom on the menu) (is not related to this PR, ill create a ticket for that)

## Checklist
